README.md

```diff
@@ -7,3 +7,11 @@ This is a suitcase subpackage for reading a particular file format.
 ```
 pip install suitcase-csv
 ```
+
+## Quick Start
+
+Examples could be found [here](http://nsls-ii.github.io/suitcase/usage.html#).
+
+## Documentation
+
+See the [suitcase documentation](https://nsls-ii.github.io/suitcase).
```

Comment:
Change the link to use the facility-independent documentation: https://blueskyproject.io/suitcase.